### PR TITLE
The API for fetching tags from a remote registry or index changed.

### PR DESCRIPTION
**Pull request by mhrivnak**
This uses the new API.

https://pulp.plan.io/issues/914

closes #914

### CODE REVIEW

File: plugins/pulp_docker/plugins/registry.py [hunk 1]
```diff
@@ -53,7 +53,16 @@ def _get_single_path(self, path):
 
         :return:    whatever gets deserialized out of the response body's json
         """
-        url = urlparse.urljoin(self.registry_url, path)
+        # if talking to docker hub, we'll get an endpoint specified, and then we'll have to get
+        # tags from that endpoint instead of talking to the original feed URL.
+        if self.endpoint:
+            # we assume the same scheme that the registry URL used
+            registry_parts = urlparse.urlsplit(self.registry_url)
```
Comment (jortel) [inline plugins/pulp_docker/plugins/registry.py]: consider *url_parts* instead of *repository_parts*.

**Comment by jortel** [comment]
LGTM

**Comment by jortel** [comment]
Looking at 'closes #914' in the description.  Will this cause redmine to close the issue when merged?  If so, this seems wrong since it's an issue and should only be closed after verified by QE.

**Comment by rbarlow** [comment]
On 05/01/2015 02:34 PM, Jeff Ortel wrote:
> Looking at 'closes #914' in the description. Will this cause redmine to
> close the issue when merged? If so, this seems wrong since it's an issue
> and should only be closed after verified by QE.

We've configured Redmine to move the issue to modified with the keyword
closes. It's not the best naming (since closes doesn't actually close
the issue), but don't worry - it'll still go through the normal process.

-- 
Randy Barlow

**Pull request merged**